package.json

```diff
@@ -17,7 +17,14 @@
   },
   "homepage": "https://github.com/Half-Shot/matrix-appservice-twitter#readme",
   "dependencies": {
-    "twitter": "^1.3.0",
-    "matrix-js-sdk": "git+ssh://git@github.com/matrix-org/matrix-js-sdk.git#develop"
+    "html-entities": "^1.2.0",
+    "js-yaml": "^3.6.1",
+    "matrix-appservice-bridge": "git+ssh://git@github.com/matrix-org/matrix-appservice-bridge.git#develop",
+    "matrix-js-sdk": "git+ssh://git@github.com/matrix-org/matrix-js-sdk.git#develop",
```

Comment:
This is dangerous. If you need to use an unreleased version, pin it to a GIT SHA instead.